compiler/optimizer/VPHandlers.cpp

```diff
@@ -7109,6 +7109,12 @@ TR::Node *constrainIshl(OMR::ValuePropagation *vp, TR::Node *node)
       vp->replaceByConstant(node, constraint, lhsGlobal);
       }
 
+   // Replace shift of constant zero with constant zero
+   if (lhs && lhs->asIntConst()
+       && lhs->asIntConst()->getInt() == 0) {
+      return vp->replaceNode(node, node->getFirstChild(), vp->_curTree);
+   }
```

Comment:
Make sure to use the style for braces that's usually used in the JIT compiler:

```
if lhs && lhs->asIntConst()
       && lhs->asIntConst()->getInt() == 0)
   {
   return vp->replaceNode(node, node->getFirstChild(), vp->_curTree);
   }
```

The same comment applies for the other `constrain` methods below.